doc/LIMITS.md

```diff
@@ -35,32 +35,36 @@ const node = await createLibp2pNode({
     /**
      * The total number of connections allowed to be open at one time
      */
-    maxConnections: number
+    maxConnections: number,
 
     /**
      * If the number of open connections goes below this number, the node
      * will try to connect to randomly selected peers from the peer store
      */
-    minConnections: number
+    minConnections: number,
 
     /**
      * How many connections can be open but not yet upgraded
      */
-    maxIncomingPendingConnections: number
-  }
-})
+    maxIncomingPendingConnections: number,
+  },
+});
+
+export {};
```

Comment:
This `export` could be confusing to end-users but I haven't found another way to avoid the error about redeclaring block-scoped variables unless we change the name of the variable, which would be required for each declaration of `node`

Reply:
Usually the `isolatedModules` controls errors in ts code that neither imports nor exports anything - did you try that?

Would rather not have to put `export {}` everywher.